py/mockbuild/buildroot.py

```diff
@@ -161,6 +161,13 @@ def _init(self, prebuild, do_log):
                 self.state.start(update_state)
                 self.pkg_manager.update()
                 self.state.finish(update_state)
+        else:
+            self._fixup_build_user()
```

Comment:
Instead of this call self.uid_manager.dropPrivsTemp() and at the enf of else block self.uid_manager.restorePrivs()